common/views/components/TabNav/TabNav.styles.tsx

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import Space from '../styled/Space';
+import { classNames } from '../../../utils/classnames';
+
+export const TabsContainer = styled.div.attrs({
+  className: 'no-margin',
```

Comment:
fwiw there are utility classes for `no-padding`, `flex`, and `plain-list` as well

Reply:
As part of my suggested work for the refactoring, since we all seemed to want to go with styled-components styling first - moving away from classes, I suggested [moving away from classes that only style one specific thing](https://github.com/wellcomecollection/wellcomecollection.org/issues/8420), so it's the direction I opt for when doing new things. Up for discussion though?